server/main.lua

```diff
@@ -0,0 +1,101 @@
+---@class vehicleData
+---@field model string
+---@field plate string
+---@field mods string
+
+---@class playerData
+---@field license string
+---@field citizenId string
+
+
+--- Creates a Vehicle DB Entity
+---@param playerData playerData
+---@param vehicleData vehicleData
+local function createEntity(playerData, vehicleData)
+    MySQL.insert('INSERT INTO player_vehicles (license, citizenid, vehicle, hash, mods, plate, state) VALUES (?,?,?,?,?,?,?)', {
+        playerData.license,
+        playerData.citizenId,
+        vehicleData.model,
+        joaat(vehicleData.model),
+        vehicleData.mods or '{}',
+        vehicleData.plate,
+        0
+    })
+end
+
+exports('CreateVehicleEntity', createEntity)
+
+---@class FetchVehicleEntityQuery
+---@field valueType 'citizenid'|'license'|'plate'
+---@field value string
+
+---@alias vehicleEntity table
+
+--- Fetches DB Vehicle Entity
+---@param query FetchVehicleEntityQuery
+---@return vehicleData[]|nil
+local function fetchEntity(query)
```

Comment:
I think a better API would be splitting this into multiple functions

i.e. FetchVehicleEntityByCitizenId, FetchVehicleEntityByLicense, FetchVehicleEntityByPlate

Reply:
And as per your previous review, you asked for this to be put into a table so we can control formatting. What formatting are we wanting for these?

Reply:
Start with the fields of the SQL table and then rename, change types, and remove duplicates